_projects/ca-child-welfare.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: project
+title: Protecting At Risk Kids
+description: Using modular contracting to help the State of California upgrade its legacy child welfare system.
+permalink: /projects/ca-child-welfare/
+tags: project
+image: /assets/img/project-images/california.jpg
```

Comment:
caption?

Reply:
Will create a separate issue for captions for images on example pages.